src/components/input-field/input-field.tsx

```diff
@@ -231,7 +231,9 @@ export class InputField {
                     class={`
                         mdc-floating-label
                         ${
-                            this.value || this.isFocused
+                            this.value ||
+                            this.isFocused ||
+                            this.mdcTextField?.value
```

Comment:
This isn't a bug that affects normal use of the component. When `this.value` is set programmatically, the label is correctly updated. When setting the value of the native input using the console, not only does the label not update, but `this.value` also doesn't update.

Since programmatically setting the value on the native input field isn't the right way to use the component, I don't think we should spend time on making the component update the value if that's done (you can probably already make it update by triggering a `change` event on the native element), and if we don't change the underlying behaviour, I don't think we should change the behaviour for the label either, as that just camouflages the fact that the value isn't updated correctly.

![setting input value through console](https://user-images.githubusercontent.com/154725/87919653-6786df80-ca78-11ea-9f20-77ae1cff324a.gif)